cypress/integration/art_view_spec.js

```diff
@@ -0,0 +1,76 @@
+const { get } = require("http")
+
+describe('check to see if it works', () => {
+  beforeEach(() => {
+    cy.intercept('https://api.artic.edu/api/v1/artworks/search?q=painting&fields=image_id,title,date_start,date_end,artist_title,thumbnail', 
+    {
+      "data": [
+        {
+          "_score": 108.59207,
+          "thumbnail": {
+            "alt_text": "A work made of oil on canvas.",
+            "width": 10184,
+            "lqip": "data:image/gif;base64,R0lGODlhBgAFAPQAAL9xNL9yOc5pH9BoH8pnJMxpJdFpJ9NsKNZtKNZuKMBxNdaPL8yeJ8yfJ82eK8ucLNOMMtaOMdaPMcGBTdOJS8ybS8CEUdCKUNaPVc6aUNaXU9iQUdWaUgAAAAAAAAAAACH5BAAAAAAALAAAAAAGAAUAAAUYoBUogDJdRCEMFGYcCbJxkLREWvU0jJOFADs=",
+            "height": 9067
+          },
+          "date_start": 1953,
+          "date_end": 1954,
+          "artist_title": "Mark Rothko",
+          "image_id": "df0a03a4-be1c-3f8d-73b1-5941dca57bba",
+          "title": "Untitled (Painting)"
+        }
+      ]
+    })
```

Comment:
Thank you for figuring out how to properly stub a response! Very much appreciated!